### DISCUSSION TITLE
Welcome to zio-zmx Discussions!

### DISCUSSION THREAD

**Discussion by softinio** [zio/zio-zmx#167] [Users]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Can ask questions as a user
  * Can ask questions as a contributor
  * Suggest new feature requests and Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## Some Useful Links
- [ZIO ZMX Documentation](https://zio.github.io/zio-zmx/)
- [ZIO ZMX Contributor's guidelines](https://zio.github.io/zio-zmx/docs/about/about_contributing)
- [ZIO Code of Conduct](https://zio.github.io/zio-zmx/docs/about/about_coc)

**Comment by atooni** [comment]
Hi there, 

I am currently using ZMX to get more into FP, especially with ZIO. For starters, I am working on PR #155, which should bring a unified API 
to capture metrics in ZIO applications and report them to one of the supported backends. 

I need ZMX as one of the features in my own project, so I thought this is a good place to get started. 

See you around ...
